packages/astro/test/astro-markdown-plugins.test.js

```diff
@@ -2,6 +2,7 @@ import { expect } from 'chai';
 import cheerio from 'cheerio';
 import { loadFixture } from './test-utils.js';
 import markdownRemark from '@astrojs/markdown-remark';
+import addClasses from '../../../examples/with-markdown-plugins/add-classes.mjs';
```

Comment:
Could you possibly move this module into the test fixture? Would be nice to not couple the tests with the examples.

Reply:
Done! I also added `hast-util-select` as a dev dependency to ensure it's installed for the tests